apps/cms/pages/templates/a4_candy_cms_pages/home_page.html

```diff
@@ -15,7 +15,7 @@ <h1 class="homepage-header__teaser p-3 ml-md-4">{{ page.teaser|richtext }}</h1>
             </div>
         </div>
 
-        <div class="p-3">{{ page.body|richtext }}</div>
```

Comment:
But now the text clings to the grey box if there is text?

Reply:
@fuzzylogic2000 I did wonder about that, we don't seem to use the normal page body for anything so having an empty padded box seemed unwise and styling something that we don't use didn't make sense either, what do you think?

Reply:
I think, we should remove the whole div in that case. Otherwise someone might use it and then it will look crappy.

Reply:
nice! will do so